pkg/api/types.go

```diff
@@ -386,12 +386,14 @@ const (
 	ClaimBound PersistentVolumeClaimPhase = "Bound"
 )
 
-// HostPathVolumeSource represents a host directory mapped into a pod.
```

Comment:
I think the struct name was used here to conform with godoc. Is this not a requirement ?